cypress/e2e/po/components/global-role-binding.po.ts

```diff
@@ -0,0 +1,12 @@
+import ComponentPo from '@/cypress/e2e/po/components/component.po';
+import CheckboxInputPo from '~/cypress/e2e/po/components/checkbox-input.po';
```

Comment:
just nitpick but shouldn't we use `@/cypress/...`?

Reply:
Good catch! Have updated both this and below